packages/account/src/Components/forms/form-fields.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
If we are creating new files then according to the process it should be in .tsx right?
Just curious

Reply:
And also a test coverage i suppose

Reply:
The base component is yet to be migrated to TSX. hence this component could not be migrated.